input/fsh/aliases.fsh

```diff
@@ -0,0 +1,2 @@
+Alias: $ncit.owl = http://purl.obolibrary.org/obo/ncit.owl
+Alias: $disease-codes = http://fhir.ncpi-project-forge.io/ValueSet/disease-codes
```

Comment:
I think curating all the aliases in one fsh file seems like a good idea! Could you move aliases in patient and specimen into this file as well? Plus, we need to talk about the naming convention for this.